examples/phosh-demo/configuration.nix

```diff
@@ -0,0 +1,129 @@
+{ config, lib, pkgs, ... }:
+
+let
+  terminal = pkgs.kgx.override { genericBranding = true; };
+
+  # One-stop shop to customize the default username before building.
+  defaultUserName = "alice";
+in
+{
+  config = lib.mkMerge [
+    # INSECURE STUFF FIRST
+    # Users and hardcoded passwords.
+    {
+      # Forcibly set a password on users...
+      # Note that a numeric password is currently required to unlock a session
+      # with the plasma mobile shell :/
+      users.users.${defaultUserName} = {
+        isNormalUser = true;
+        # Numeric pin makes it **possible** to input on the lockscreen.
+        password = "1234";
```

Comment:
How about `users.users.<name>.hashedPassword` instead?

Reply:
It's not meant for actual use. This is meant more of an example configuration you can look at as a starting point.

Including this in your system configuration is foolish :).

We probably should make this more obvious in some way.

Reply:
Had some clashes with user names and identifiers while upgrading profile from the xfce demo. Had to change user name to "nixos" and id to 1001 (in several places) to make it start.